### ISSUE TITLE
Crash saving order: AttributeError: 'NoneType' object has no attribute 'financecompany'

### ISSUE THREAD

**Issue by kentbower** [kbower/tickettest1#2023]
_Imported from trac ticket 2023.
Created by **kbower**
Opened in trac: Nov 13, 2012, 8:37:51 AM
Last modified in trac: Nov 13, 2012, 10:14:40 AM_
***
_by kbower:_
self.paymenttypeid is apparently None
```python
#if payment type is a 3rd party funding payment type, ensure user has permission to use it
if self.paymenttype.financecompany and \
   not Permission.user_has('paythirdparty'):
    self.error('{ArTran.thirdpartyprequired}','amount')
```

**Comment by kentbower** [comment]
_**kbower** trac comment on Nov 13, 2012, 8:39:27 AM:_

I think the system was trying to do dollar transfers.

**Comment by kentbower** [comment]
_**kbower** trac comment on Nov 13, 2012, 8:47:03 AM:_

Also important: please check if that error shows up from both screens, order and customer... I'm suspicious that it might be broken.

**Comment by kentbower** [comment]
_**rneville** trac comment on Nov 13, 2012, 10:14:40 AM:_

(Changeset [[3384]](../commit/e2c2164c98472e97d71a20391c33c01d03faae25))
* fixes #2023 (Crash saving order: AttributeError: 'NoneType' object has no attribute 'financecompany')

**Comment by kentbower** [comment]
_**rneville** trac status change on Nov 13, 2012, 10:14:40 AM:_ **closed**
